app/src/main/java/com/myskng/virtualchemlight/activity/UOActivity.kt

```diff
@@ -16,10 +16,7 @@ import android.view.*
 import com.myskng.virtualchemlight.R
 import com.myskng.virtualchemlight.uo.UOSensor
 import com.myskng.virtualchemlight.databinding.ActivityUoBinding
-import kotlinx.coroutines.CoroutineScope
-import kotlinx.coroutines.Dispatchers
-import kotlinx.coroutines.SupervisorJob
-import kotlinx.coroutines.launch
+import kotlinx.coroutines.*
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/tumugin/VirtualChemLight/blob/73b2b1974b36173d0c10b38515cdc11ccd0260a0/app/src/main/java/com/myskng/virtualchemlight/activity/UOActivity.kt#L19"></span>Wildcard import (cannot be auto-corrected)</td>
    </tr>
  </tbody>
</table>